### PR TITLE
Add feature to open in editor instead of Finder

### PR DESCRIPTION
**Pull request by thibmaek**
Setting openInEditor in the config will make clicking on the cwd open in an editor of your choice instead.

### CODE REVIEW

**Comment by thibmaek** [comment]
Can I do something else to get this merged?

File: index.jsx [hunk 1]
```diff
@@ -335,7 +347,18 @@ exports.decorateHyper = (Hyper, { React }) => {
         }
 
         handleCwdClick(event) {
-            shell.openExternal('file://' + this.state.cwd);
+            if (globalConfig.hyperStatusLine.openInEditor) {
```
Comment (j-f1) [inline index.jsx]: How do you feel about getting rid of the `openInEditor` option and just testing if `editor` is provided?

File: index.jsx [hunk 2]
```diff
@@ -322,6 +323,17 @@ const setGit = repo => {
...
 };
 
+const getEditor = editorType => {
+    switch (editorType) {
+        case 'vscode':
+            return '/usr/local/bin/code';
+        case 'atom':
+            return '/usr/local/bin/atom';
+        default:
+            return 'open';
```
Comment (j-f1) [inline index.jsx]: Should this be a default? IIRC `open` isn’t available on Linux.

**Review by j-f1 — CHANGES_REQUESTED** [review]
A couple questions:

File: index.jsx (hunk 2 above)
Comment (thibmaek) [reply index.jsx]: Hmm I'm not really sure how to determine what the default action should be.
In that case I also don't think /usr/local/bin would work on Windows & Linux

File: index.jsx (hunk 1 above)
Comment (thibmaek) [reply index.jsx]: Sounds good.

File: index.jsx (hunk 2 above)
Comment (j-f1) [reply index.jsx]: If you merge the two options as suggested above, you could throw if there’s an invalid truthy value provided.

**Comment by thibmaek** [comment]
Hi! Sorry this took so long but I think I resolved all the remarks.

**Comment by thibmaek** [comment]
Does this look better?

**Review by j-f1 — APPROVED** [review]
I’ve been really busy recently. I might not get to publish this for a few days, but you can always install directly from GitHub instead of via npm.

**Comment by thibmaek** [comment]
Ok thank you! 🙇

**Pull request merged**